tests/lib/rules/no-insecure-url.js

```diff
@@ -166,11 +201,48 @@ ruleTester.run(ruleId, rule, {
                     );
                 };
             `,
+            output: `
+                const someSvg: React.FC = () => {
+                    return (
+                        <svg someOtherAttribute="https://ban-example.com/">
+                        </svg>
+                    );
+                };
+            `,
             errors: [
                 { messageId: "doNotUseInsecureUrl", line: 4},
             ],
             parser: testUtils.tsParser,
             parserOptions: testUtils.tsReactParserOptions,
         },
+        {
+            // should fix url correctly
+            code: `var a1 = "http://moz\ti\tlla.org";`,
+            output: `var a1 = ${JSON.stringify("https://moz\ti\tlla.org")};`,
```

Comment:
Can you write it out explicitly? shouldn't it just be `var a1 = "https://moz\ti\tlla.org";`?